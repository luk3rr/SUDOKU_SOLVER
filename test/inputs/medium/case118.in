205000490 080000731 000000000 003000200 540008960 009003005 004290000 300070000 001006070 
