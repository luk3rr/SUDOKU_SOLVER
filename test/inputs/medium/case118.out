Solving the following grid:
2 0 5 | 0 0 0 | 4 9 0 
0 8 0 | 0 0 0 | 7 3 1 
0 0 0 | 0 0 0 | 0 0 0 
------+-------+------
0 0 3 | 0 0 0 | 2 0 0 
5 4 0 | 0 0 8 | 9 6 0 
0 0 9 | 0 0 3 | 0 0 5 
------+-------+------
0 0 4 | 2 9 0 | 0 0 0 
3 0 0 | 0 7 0 | 0 0 0 
0 0 1 | 0 0 6 | 0 7 0 


Solution found :')

2 3 5 | 1 8 7 | 4 9 6 
4 8 6 | 9 5 2 | 7 3 1 
1 9 7 | 3 6 4 | 8 5 2 
------+-------+------
6 1 3 | 5 4 9 | 2 8 7 
5 4 2 | 7 1 8 | 9 6 3 
8 7 9 | 6 2 3 | 1 4 5 
------+-------+------
7 6 4 | 2 9 5 | 3 1 8 
3 5 8 | 4 7 1 | 6 2 9 
9 2 1 | 8 3 6 | 5 7 4 

Algorithm: BFS
Total time: 604 ms
Total expanded states: 93242
